global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "7.0.306",
+    "version": "7.0.400",
```

Comment:
We should hold off on the .NET version bump a couple of days to keep the SF and .NET changes separated in downstream automated package updates.

Reply:
These are also being pushed out by the auto package updates, I'd say to hold off on the SF change until the .NET change has made it all the way as .NET is harder to gate.

Reply:
It's already being bumped elsewhere, may as well let it go through.